### ISSUE TITLE
steamtricks seems to break when using the binary NVidia driver

### ISSUE THREAD

**Issue by rombert** [steamtricks/steamtricks#40]
I've noticed that since the steam update that brought in the ability to use native libraries steam no longer works. I am using an NVidia GTX 1070 video card with driver 375.39. This also happened when I was using an NVidia GTX 750Ti video card.

On Tumbleweed, 

    export NO_STEAMTRICKS=1

works for me.

Another user has reported in the openSUSE forums that removing steamruntime-fix has fixed a similar issue ( I assume that's on Leap ). The user is using a Geforce 780 with driver 375.39.

On the other hand, on my laptop using Intel graphics the steamtricks integration does not cause any problems.

I'm raising this for discussion, I'm not sure whether this affects all nvidia users of just a subset and whether something should be done in this project.

**Comment by rombert** [comment]
See also https://bugzilla.opensuse.org/show_bug.cgi?id=1032151

**Comment by boombatower** [comment]
I responded on bug, if someone can provide a conditional that is true when nvidia binary driver is active then can simple disable the openssl fix.

**Comment by boombatower** [comment]
A response to another bug suggested:
```bash
lsmod | grep nvidia
```

That will indicate if the binary driver is installed, but not if it is running (unless the status code is parsed). Perhaps another command that indicates if loaded. Otherwise, a glxinfo based solution may be better to indicate that it is being used for the current display.

**Comment by boombatower** [comment]
The references to Mesa that are likely different when using nvidia binary driver.
```
glxinfo | grep Mesa
client glx vendor string: Mesa Project and SGI
OpenGL core profile version string: 4.5 (Core Profile) Mesa 17.0.1
OpenGL version string: 3.0 Mesa 17.0.1
OpenGL ES profile version string: OpenGL ES 3.1 Mesa 17.0.1
```

The 1st or 3rd seem the like the best to use with preference on the 1st.

**Comment by boombatower** [comment]
https://github.com/steamtricks/steamtricks/compare/nvidia-binary-openssl-disable

If I change the grep to look for `radeon` it disables openssl replacement. Can you potentially dump the glxinfo or see if a better method of detecting the nvidia binary driver.

If you would rather the simplest method to test if just disabling openssl replacement works just change `/usr/bin/steamtricks` to `STEAM_OPENSSL_REPLACE=0` near the top.

**Comment by boombatower** [comment]
Another seemingly related bug https://bugzilla.opensuse.org/show_bug.cgi?id=1021095.

**Comment by rombert** [comment]
Tested with `STEAM_OPENSSL_REPLACE=0` and it works fine - launched X-Com 2 and Serious Sam 3. 
I've also attached the output of glxinfo in case that helps.

[glxinfo.txt](https://github.com/steamtricks/steamtricks/files/1011303/glxinfo.txt)

**Comment by boombatower** [comment]
Thanks for trying that out. I used your glxinfo to make a check and falls back to lsmod if glxinfo is not available rather than make it a requirement just for this.

Fixed in 5b9bba38db1b005a772cf34d21ee32b07efbf0f4.

**Comment by rombert** [comment]
Excellent, thank you.
